data/brands/shop/bakery.json

```
@@ -386,7 +386,7 @@
     {
       "displayName": "Brioche Dorée",
       "id": "briochedoree-731a34",
-      "locationSet": {"include": ["fr"]},
+      "locationSet": {"include": ["001"]}
```

try to remember the comma..
```suggestion
      "locationSet": {"include": ["001"]},
```